civis/ml/_model.py

```diff
@@ -527,6 +539,15 @@ class ModelPipeline:
     notifications : dict
         See :func:`~civis.resources._resources.Scripts.post_custom` for
         further documentation about email and URL notification.
+    dependencies : array, optional
+        List of packages to install from PyPI or Github. If a private Github
```

Comment:
This should be more general than just GitHub, e.g. https://stackoverflow.com/questions/38335377/install-python-package-from-private-bitbucket-repo .